server/service/handler.go

```diff
@@ -754,6 +754,8 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	// input to `fleetctl apply`
 	ue.POST("/api/_version_/fleet/mdm/profiles/batch", batchSetMDMProfilesEndpoint, batchSetMDMProfilesRequest{})
 
+	ue.GET("/api/_version_/fleet/ota", getOTAProfileEndpoint, getOTAProfileRequest{})
```

Comment:
In the spec I went with `/api/v1/fleet/enrollment_profiles/ota` to make it consistent with the automatic and manual profile endpoint, could you sanity check with the API DRI before merging?

Reply:
@roperzh that's a better path IMO, I just goofed in noticing that change in the API spec you linked

Reply:
w00t! I asked Luke to give us a definitive answer, we'll se what he says